Ethnic/ETH1219baryas.xml

```diff
@@ -36,17 +36,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-10-18"> Created file from google
                                 spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: ethnic</change>
+         <change when="2024-09-17" who="CH">Updated name and added bibliography.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <personGrp role="ethnic">
-               <persName xml:id="n1">
-                  <orgName>barya, soc. group</orgName>
-               </persName>
+               <persName xml:id="n1" xml:lang="gez"><orgName>ባርያ፡</orgName></persName>
+               <persName corresp="#n1" xml:lang="gez" type="normalized"><orgName>Bāryā (outdated idiom)</orgName></persName>
             </personGrp>
```

Comment:
remove `outdated`, if necessary add more helpful information on what the term implies (something like 

`a name previously used for <persName ref="ETH1946Nara"/>, later for a broader group of "blacks", a social group, similar to <persName ref="ETH2043sanqel"/>. later perceived as pejorative - as reflected by the <ref type="authFile" corresp="Barya">name for the demonic entity</ref>` )

etc